cmd/ocm-backplane/console/console_suite_test.go

```diff
@@ -1,13 +0,0 @@
-package console
-
-import (
-	"testing"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-func TestIt(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Console Test Suite")
-}
```

Comment:
Can you comment it out instead of removing it entirely? With the follow-up Jira as the comment header.